### PR TITLE
Added --name to docker run command

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
This PR will resolve issue #7 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,5 +79,5 @@ class User(Base):
 
 ```
 >>> sudo docker image build .
->>> sudo docker run -d -p 8080:80 <image>
+>>> sudo docker run -d --name -p 8080:80 <image>
```
Comment (abhayaman669) [inline README.md]: change `--name` to `--name myapp`

Comment (96RadhikaJadhav) [reply README.md]: Is this fine? -> sudo docker run -d --name myapp  8080:80 <image>

Comment (abhayaman669) [reply README.md]: Yes 👍

Comment (96RadhikaJadhav) [reply README.md]: Done.

**Comment by abhayaman669** [comment]
Thanks @96RadhikaJadhav 😊

**Pull request merged**